mixin/README.md

```diff
@@ -59,48 +59,54 @@ This project is intended to be used as a library. You can extend and customize d
 [embedmd]:# (config.libsonnet)
 ```libsonnet
 {
+  local thanos = self,
+  // Hierarcies is a way to help mixin users to add high level hierarcies to their alerts and dashboards.
```

Comment:
WDYT `groupby` or `groups` or `tagets`?

```suggestion
  // Groups <slice?> is a way to help mixin users to add high level target grouping to their alerts and dashboards.
```

Reply:
Actually, I don't understand why hierarchies are helpful here and why we need them? Can we just show in an example `dimensions/aggregators: job, namespace, cluster`?